srcpkgs/base-files/files/_vkpurge

```diff
@@ -0,0 +1,29 @@
+#compdef vkpurge
+
+_vkpurge() {
```

Comment:
Adding function definition is redundant in this case. I'm suggesting to remove function definition and decrease indentation by one level. Or add `_vkpurge "$@"` at the end of the file to support autoloading in some emulation mode.